src/content_sao.cpp

```diff
@@ -118,15 +118,14 @@ UnitSAO::UnitSAO(ServerEnvironment *env, v3f pos):
 	m_armor_groups["fleshy"] = 100;
 }
 
-bool UnitSAO::isAttached() const
+ServerActiveObject *UnitSAO::getParent() const
 {
 	if (!m_attachment_parent_id)
-		return false;
+		return nullptr;
 	// Check if the parent still exists
 	ServerActiveObject *obj = m_env->getActiveObject(m_attachment_parent_id);
-	if (obj)
-		return true;
-	return false;
+
+	return obj;
```

Comment:
Should this check 'if (obj)' as before?

Reply:
It's now a different function. The name and return type has changed.

Reply:
Ok i see.